jobs/uaa-customized/templates/resources/javascripts/pw_requirements.js

```diff
@@ -2,9 +2,9 @@ $(document).ready(function(){
   // Get all the password policy requirements.
   // In case the policy equals to null, initialize the values to 0.
   var lengthCount = <%= p("uaa.password.policy.minLength") %>;
-  var passwordField = document.getElementsByName("password")[0].value || "password";
-  var confirmPasswordField = document.getElementsByName("password_confirmation")[0].value || "password_confirmation";
-  var submitButton = document.getElementsByName("submit")[0].value || "submit";
+  var passwordField = "password";
```

Comment:
That part of the code did work, but isn't actually necessary. We don't want the actual value of any of these fields, just the name so that it can be passed to the validate password function.